bash/02_build.sh

```diff
@@ -11,16 +11,14 @@ docker run --rm\
     -w /home/colp_build\
     -u $(id -u ${USER}):$(id -g ${USER}) \
     -e BUILD_ENGINE=$BUILD_ENGINE\
-    nycplanning/docker-geosupport:latest bash -c "python3 -m python.geo_qaqc;
-                                                python3 -m python.geocode"
+    nycplanning/docker-geosupport:latest bash -c "python3 -m python.geocode"
 
 psql $BUILD_ENGINE -f sql/_procedures.sql
 psql $BUILD_ENGINE -f sql/clean_parcelname.sql
 psql $BUILD_ENGINE -f sql/create_colp.sql
-psql $BUILD_ENGINE -f sql/geo_qaqc.sql
```

Comment:
I think moving this out of the build step creates an issue on my local machine @SashaWeinstein. wanna swing by when you get a chance?